translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+    "name": "csatlakozás",
+    "description": "Csatlakozol a szerveren futó eseménybe.",
+    "options": [],
+    "responses": {
+      "eventJoined": [
+        "Sikeresen csatlakoztál az eseménybe a **{ctx.guild.name}** nevű guild színeiben!"
+      ]
+    }
+  },
+  "event leave": {
+    "name": "elhagyás",
+    "description": "Elhagyod a szerveren futó eseményt.",
+    "options": [],
+    "responses": {
+      "eventLeft": [
+        "Sikeresen elhagytad az eseményt a **{ctx.guild.name}** nevű guild színeiben!"
+      ]
+    }
+  },
+  "event position": {
+    "name": "helyezés",
+    "description": "Mutatja egy játékos helyezését az eseményben.",
+    "options": [
+      {
+        "name": "játékos",
+        "description": "A játékos, akinek a helyezését szeretnéd."
+      }
+    ],
+    "responses": {
+      "notJoined": [
+        "{player} nincs az eseményben."
+      ],
+      "playerPosition": [
+        "{player}-nek {xp:,} {label}-je van az eseményben és jelenleg a #{pos:,} helyen áll a {players:,} játékosból."
+      ]
+    }
+  },
+  "event list": {
+    "name": "lista",
+    "description": "Mutatja az összes játékost az eseményben.",
+    "options": [],
+    "responses": {
+      "eventNoMembers": [
+        "Senki sincs az eseményben!"
+      ],
+      "title": [
+        "Esemény Ranglista"
+      ],
+      "eventMembers": [
+        "**Total Participants**: {players:,}\n\n",
+        "{data}"
+      ]
+    }
+  },
+  "event guild": {
+    "name": "guild",
+    "description": "Megnézi hogyan haladnak egy guild tagjai az eseménnyel.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A keresnivaló guild."
+      }
+    ],
+    "responses": {
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "title": [
+        "A {guild} nevű guild Eseményadatai"
+      ],
+      "guild": [
+        "A **{guild}** nevű guildnak {members} tagja vesz részt az eseményen és együtt átlagolnak {avg:,} {label}-t tagonként. ",
+        "(Total {label}: {total:,})"
+      ],
+      "noMembers": [
+        "A **{guild}** nevű guild-nak nincsenek tagjai az eseményen."
+      ]
+    }
+  },
+  "lookup": {
+    "name": "kikeresés",
+    "description": "Megkeresi, hogy a játékos egy scammer, IRL kereskedő, rattelő vagy makrózó."
+  },
+  "lookup player": {
+    "name": "játékos",
+    "description": "Megkeresi, hogy valaki egy scammer, IRL kereskedő, rattelő vagy makrózó. Megnézi a Discord fiókjukat, ha kötve van hozzá.",
+    "options": [
+      {
+        "name": "játékos",
+        "description": "A megnézendő játékos."
+      }
+    ],
+    "responses": {
+      "notScammer": [
+        "**{player}** nincs megjelölve scammerként/IRL kereskedőként az adatbázisunkban.\n",
+        "Ez ugyanakkor nem jelenti azt, hogy százszázalékosan nem scammel, mindig légy óvatos, amikor más játékosokkal cserélsz.",
+        "\n\n*Ha ez nem igaz, kérjük jelentsd ezt a játékost `/report` paranccsal.*"
```

Comment:
The command name should be translated
Should also be in backticks for `this effect`